### PR TITLE
Add PyPi classifiers in setup.py

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -77,6 +77,18 @@
...
+
+    'classifiers': [
+        'Development Status :: 6 - Mature',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```
Comment (mgeplf) [inline setup.py]: do we need to specify 3.6?

Comment (wizmer) [reply setup.py]: Good question. I don't know. In this [example](https://packaging.python.org/guides/distributing-packages-using-setuptools/#classifiers) they seem to specify each version. Maybe it's best to stay vague...

Comment (mgeplf) [reply setup.py]: I think you need to specify all versions that are supported - in the example, they don't support 3.1, for instance....

**Comment by wizmer** [comment]
For reference, numpy seem to only care about python 3.5, 3.6, 3.7, here are their classifiers:
```

CLASSIFIERS = """\
Development Status :: 5 - Production/Stable
Intended Audience :: Science/Research
Intended Audience :: Developers
License :: OSI Approved
Programming Language :: C
Programming Language :: Python
Programming Language :: Python :: 3
Programming Language :: Python :: 3.5
Programming Language :: Python :: 3.6
Programming Language :: Python :: 3.7
Programming Language :: Python :: 3 :: Only
Programming Language :: Python :: Implementation :: CPython
Topic :: Software Development
Topic :: Scientific/Engineering
Operating System :: Microsoft :: Windows
Operating System :: POSIX
Operating System :: Unix
Operating System :: MacOS
"""
```

**Pull request merged**